lib/tasks/data/reindex_explainers.rake

```diff
@@ -0,0 +1,60 @@
+namespace :check do
+    namespace :data do
+      # bundle exec rake check:data:reindex_explainers
```

Comment:
When moving this file to lib/tasks/migrate, please update the namespace and comment accordingly.